[IMP] stock_by_warehouse, stock_by_warehouse_sale: Backport of warehouse widget modal.

JVegaB opened the pull request:
- The modal `stock_by_warehouse.ProductWarehousePopOver` used in the `widget="warehouse"` defined by `stock_by_warehouse`  was improved to use the version for 17.0.
- The view stock_by_warehouse_purchase.view_order_form` was improved to correctly use bootstrap alerts.

This is a backport of these commits

https://github.com/Vauxoo/addons-vauxoo/commit/23c94e67f2bbbcbc064e876ead094bdb80e413d6
https://github.com/Vauxoo/addons-vauxoo/commit/02b2b10923d4718432614fb768a05d2b2cc55569

Before:

<img width="307" height="341" alt="image_2025-08-11_14-33-30 (2)" src="https://github.com/user-attachments/assets/3b390804-e6bf-4065-96db-b7003bc59356" />

After

<img width="307" height="341" alt="image_2025-08-11_14-33-30 (2)" src="https://github.com/user-attachments/assets/55712f83-6307-4f6c-be7c-e771c671f6a1" />



Task: Inventario (#29941).

JVegaB committed:
[IMP] stock_by_warehouse: Backport of warehouse widget modal.

- The modal `stock_by_warehouse.ProductWarehousePopOver` used in the `widget="warehouse"`
  defined by `stock_by_warehouse`  was improved to use the version designed for 17.0.

AndreaLA6 commented:
+1 functional @hugho-ad

hugho-ad commented:
@JVegaB 

pls attach some screenshots, and reference the commit your are back porting please

JVegaB commented:
@hugho-ad Comments ready

hugho-ad commented:
@luisg123v 
could you merge?

luisg123v reviewed (changes requested):
Regarding commit presentation:
- Use one commit per module, please. Be aware that, when the modules are migrated, that's done for each module, retrieving the history for this module. If you touch seveal modules in the same commit, then the commit message will be duplicated when the modules are migrated
- When there are multi-line items, text of next lines should be indented, e.g.:
  
  ```
  - This is a very long multi-line item that does not fit in
    just one line
  ```
- Remove the task paragraph.

zulyvauxoo commented:
@JVegaB @hugho-ad LGTM tested in coinsamatik https://imgur.com/ZNqnUkf

hugho-ad commented:
@luisg123v 

could you please merge it?

luisg123v reviewed (changes requested):
Review https://github.com/Vauxoo/addons-vauxoo/pull/1705#pullrequestreview-3161984787 mentions several points, but the following ones are not applied yet:
- Format multi-line list items correctly
- Remove task paragraph

Besides, one of the commit messages mentions 17.0, but this is for 16.0:

"... was improved to use the version for 17.0."

JVegaB commented:
@luisg123v Commit messags improved

JVegaB committed:
[IMP] stock_by_warehouse_sale: Using correct bootstrap alerts

- The view stock_by_warehouse_purchase.view_order_form` was improved to correctly use
  bootstrap alerts.
